Zoo/Zoo/Zoo.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+namespace Zoo
+{
+    public class Zoo 
+    {
+        private string nameZoo;
+        private string[] listAnimals = new string [5];
+
+        private string nameAnimal;
+        private string nicknameID;
+        private string favoriteFood;
+
+        public static int count;
+
+        //private DateTime arrivalDate;
+        //private DateTime departureDate;
+        
+        public string NameZoo
+        {
+            get { return nameZoo; }
+            set { nameZoo = value; }
+        }
+
+        public string[] ListAnimals
+        {
+            get { return listAnimals; }
+            set { listAnimals = value; }
+        }
+
+        public string NameAnimal
+        {
+            get { return nameAnimal; }
+            set { nameAnimal = value; }
+        }
+
+        public string NicknameID
+        {
+            get { return nicknameID; }
+            set { nicknameID = value; }
+        }
+
+        public string FavoriteFood
+        {
+            get { return favoriteFood; }
+            set { favoriteFood = value; }
+        }
+        
+
+        static void AddAnimal()
+        {
+            count = 0;
```

Comment:
nu e nevoie de asta. by default e 0. daca vrei poti faci o proprietate, din care sa numeri cate animale sunt in lista de animale din zoo